pom.xml

```
@@ -93,6 +93,23 @@
             <artifactId>spring-security-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-api</artifactId>
+            <version>0.12.5</version>
```

same for other jwt dependencies
```suggestion
            <version>${jjwt.version}</version>
```